resources/compass-runtime-agent/templates/deployment.yaml

```diff
@@ -41,8 +41,6 @@ spec:
               value: "{{ .Values.compassRuntimeAgent.certificates.caCertificate.secret.namespace }}/{{ .Values.compassRuntimeAgent.certificates.caCertificate.secret.name }}"
             - name: APP_INSECURE_CONNECTOR_COMMUNICATION
               value: "{{ .Values.compassRuntimeAgent.connector.insecureCommunication }}"
-            - name: APP_INTEGRATION_NAMESPACE
```

Comment:
Please update README.md to reflect this change. 
And one additional remark here. I found in the code one place where "kyma-integration" namespace is hardcoded (clusterassetgrouprepository.go). This is probably redundant as resources created have cluster scope. We could change that but at a later time.

Reply:
Readme.md updated - removed this parameter